.github/scripts/common/lib.sh

```diff
@@ -369,7 +369,7 @@ function relative_parent() {
 # used as Github Workflow Matrix. This call is exposed by the `scan` command and can be used as:
 # podman run --rm -it -v /.../fellowship-runtimes:/build docker.io/chevdor/srtool:1.70.0-0.11.1 scan
 function find_runtimes() {
-    libs=($(git grep -I -r --cached --max-depth 20 --files-with-matches 'construct_runtime!' -- '*lib.rs'))
+    libs=($(git grep -I -r --cached --max-depth 20 --files-with-matches '[frame_support::runtime]!' -- '*lib.rs'))
```

Comment:
I think we only moved over Rococo and Westend to the new syntax. For example the rococo-bridge-hub still uses the old one.  
Not sure if that one is important, otherwise maybe keep both?

Reply:
Yep, I've tested it locally, and it gave all the runtimes as well, so I went for this one. If it is finem, I can keep it like this